src/views/portfolio/tags/TaggedProjectListModal.vue

```diff
@@ -46,13 +52,33 @@ export default {
       });
     },
   },
+  mounted() {
+    // NB: Because this modal is loaded dynamically from TagList,
+    // this.$refs.table may still be undefined when mounted() is called.
+    // https://jefrydco.id/en/blog/safe-access-vue-refs-undefined
+    const interval = setInterval(() => {
+      if (this.$refs.table) {
+        this.$refs.table.refreshOptions({
+          showBtnDeleteSelected: this.isPermitted(
+            this.PERMISSIONS.PORTFOLIO_MANAGEMENT,
+          ),
```

Comment:
`isPermitted` should now take an array instead, and check for `PORTFOLIO_MANAGEMENT` or `PORTFOLIO_MANAGEMENT_UPDATE` in accordance with the [required permissions of the REST endpoint](https://github.com/DependencyTrack/hyades-apiserver/blob/f060f75a6aff32d88a0bd389ec2c5768989d7786/src/main/java/org/dependencytrack/resources/v1/TagResource.java#L254).

For reference, see for example: https://github.com/DependencyTrack/hyades-frontend/blob/a41ecdb4585f9093929634f0916a721fe0581e7b/src/views/portfolio/projects/FindingAudit.vue#L134-L137